src/main/resources/Documentation/about.md

```diff
@@ -1,41 +1,55 @@
-TODO: Add Plugin introduction
+Eiffel Gerrit plugin is may be used to send Eiffel events when changes occurs in in a project in Gerrit.
```

Comment:
one of is/may can be removed from this sentence :)

Reply:
'occur' can be in singular.

Reply:
Done in next commit.